packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -865,12 +865,12 @@ export function TransferPanelMain({
           {showAdvancedSettings && (
             <>
               <div className="mt-2">
-                <span className="text-md text-gray-10">
+                <span className="text-md text-gray-dark">
                   Destination Address
                   {!isSmartContractWallet ? ' (optional)' : ''}
                 </span>
                 <input
-                  className="mt-1 w-full rounded border border-gray-6 px-2 py-1"
+                  className="mt-1 w-full rounded-lg border border-gray-dark px-2 py-1"
```

Comment:
According to the notion's screenshot of the Figma mockup this one should have a box shadow too

Reply:
Updated with `shadow-input`